.travis.yml

```diff
@@ -29,8 +29,9 @@ bundler_args: --without changelog development docgen guard maintenance omnibus_p
 
 matrix:
   include:
-    - rvm: 2.2
-    - rvm: 2.1
+    - rvm: 2.3.1
+    - rvm: 2.2.5
+    - rvm: 2.1.9
```

Comment:
actually let me amend that...   the RFC of mine gives us the leeway to drop support for 2.1 now that it is in security maintenance mode.  because of all the gems dropping support for 2.1 i've aggressively exercised that option and the next version of berkshelf will add support for ruby 2.3 and 2.4 while dropping support for 2.1 via the required_ruby_version in the gemspec.   so i think your hand is pretty forced here and we're deprecating 2.1 and you should bump the required_ruby_version at the same time as adding 2.3 support.